### ISSUE TITLE
Wheels and distribution files

### ISSUE THREAD

**Issue by diegoasua** [portugueslab/sashimi#78]
It might be a good idea to register the package in Pypi so that no-one other has the idea meanwhile to register another package with the same funny name, even if some things have to still be moved to scopecousine. Happy to do it if there is green light for this.

**Comment by vilim** [comment]
I don't think it is a high priority to get it into PyPI. The software as it is now is usable on exactly one setup (unless someone also has the same combination of laser, NI Board and camera), so for this purpose an git install is fine (also assuming the first users with slightly different setups will help with development).

**Comment by diegoasua** [comment]
This was not meant in relation to the ease of installation, but "booking" the name in Pypi

**Comment by vigji** [comment]
agreed on booking the name, you never know :)

**Comment by vilim** [comment]
Ok, than could you please Luigi set up the autodeployment (maybe also the docs while you are at it)

On 7 Aug 2020 22:45, Luigi Petrucco <notifications@github.com> wrote:
August 7, 2020 10:45:41 PM CEST Luigi Petrucco <notifications@github.com> wrote:

agreed on booking the name, you never know :)
